manifests/kustomize/base/metadata/metadata-configmap.yaml

```diff
@@ -8,3 +9,24 @@ data:
   mysql_database: "metadb"
   mysql_host: "mysql"
   mysql_port: "3306"
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: metadata-mysql-configmap
+  labels:
+    component: metadata-server
+data:
+  mysql_database: "metadb"
```

Comment:
As discussed let's maybe make the keys upper case to be consistent

Reply:
Done.